addon/mixins/taggable-mixin.js

```diff
@@ -8,7 +8,7 @@ export default Ember.Mixin.create({
             Ember.A(currentTags);
```

Comment:
Right, but I think you've made two copies, and this second one appears unused. Does it work without this line?

(both `slice` and `Ember.Array` should return a new copy)